vpnglean/glean_parser_ext/templates/cpp.jinja2

```diff
@@ -38,28 +38,31 @@ struct {{ obj.name|Camelize }}{{ suffix }} {
   {% endfor %}
 
   FfiExtra ToFfiExtra() const {
-    QVector<const char*> extraKeys;
-    QVector<const char*> extraValues;
+    const char* extraKeys[{{ obj|attr(name)|length }} + 1];
+    const char* extraValues[{{ obj|attr(name)|length }} + 1];
```

Comment:
Is it an issue that I create the array as large as it can be, but then calculate the actual count of elements and that is what I tell Rust is the actualy size of this array? (Sorry if this doesn't make sense)